src/stactools/cop_dem/__init__.py

```diff
@@ -8,4 +8,4 @@ def register_plugin(registry):
     registry.register_subcommand(commands.create_cop_dem_command)
 
 
-__version__ = "0.1.0"
+__version__ = "0.1.1"
```

Comment:
```suggestion
__version__ = "0.1.0"
```

Let's not bump the version yet -- IMO it's best to release off of a standalone PR, so it's easier to audit the changed and ensure (e.g.) there is nothing breaking that would require a v0.2.

Reply:
Sounds good.